### PR TITLE
Review Response of #17

### PR DESCRIPTION
**Pull request by mactkg**
This PR is response of https://github.com/mactkg/vscode-sonic-pi/pull/17 .
If you have a question, feel free to leave comment to this PR.

Thanks!

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import * as vscode from 'vscode';
 
+const BACKGROND_COLOR_FLASHED = 'rgba(255,20,147,1.0)'
```
Comment (mogimogitomato) [inline src/config.ts]: It is a light point out, but `BACKGROND` spell incorrect.

**Review by mogimogitomato — COMMENTED** [review]
Thank you, I commented one point. Best regards.

File: src/config.ts (hunk 1 above)
Comment (mactkg) [reply src/config.ts]: oops!

**Comment by mactkg** [comment]
Thanks, fixed!

**Pull request merged**